.github/workflows/greeting.yml

```diff
@@ -0,0 +1,24 @@
+name: Greeting github issue
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "OfficeDev/teamsfx-issue-triage-bot"
+          ref: main
+          path: ./actions
+          token: ${{secrets.TEAMSFX_BOT}}
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+
+      - name: Run Greeting 
+        uses: ./actions/greeting
+        with:
+          issue-message: "Thank you for contacting us! Any issue or feedback from you is quite important to us. We will do our best to fully respond to your issue as soon as possible. Sometimes additional investigations may be needed, we will usually get back to you within 2 days by adding comments to this issue. Please stay tuned."  
```

Comment:
"within 2 business days"?
By the way, what's the display name of this bot?

Reply:
the message follows pm decision. 
the display name is 
![image](https://user-images.githubusercontent.com/16380704/125724163-57b70b6d-157a-4a6e-a16d-ce76502c8eb8.png)